dimod/bqm/shapeablebqm.pyx.src

```diff
@@ -914,6 +914,52 @@ cdef class cyAdj@name@BQM:
 
         self.bqm_.set_linear(vi, b)
 
+    def add_constraint_as_quadratic(self, object terms, Bias
```

Comment:
This method sounds lot more general than it is.

Call it perhaps `add_linear_equality_constraint`? Or `add_linear_equality_constraint_as_penalty`?

Reply:
We would need to also update the DQM version. I am not necessarily opposed (it hasn't been released yet). Of the two, I prefer `add_linear_equality_constraint`. But would defer to @hsadeghidw ultimately.

Reply:
good call. The name should change, also for DQM solver.